openBCIBoard.js

```diff
@@ -988,16 +987,62 @@ function OpenBCIFactory() {
     };
 
     /**
+     * @description Start logging to the SD card.
+     * @param recordingDuration {String} - The duration you want to log SD information for. Limited to:
+     *      '14sec', '5min', '15min', '30min', '1hour', '2hour', '4hour', '12hour', '24hour'
+     * @returns {Promise} - Resolves if the command was added to write queue.
+     * @author AJ Keller (@pushtheworldllc)
+     */
+    OpenBCIBoard.prototype.sdStart = function(recordingDuration) {
```

Comment:
do you do a check for whether the recordingDuration match the requirements above? in the docstring, it mentioned that it can only be certain value, right?